services/opm/opm-version.service.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+const { addv } = require('../text-formatters')
+const { BaseService, NotFound, InvalidResponse } = require('..')
+
+module.exports = class OpmVersion extends BaseService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'opm/v',
+      pattern: ':user/:moduleName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'OPM',
+        namedParams: {
+          user: 'openresty',
+          moduleName: 'lua-resty-lrucache',
+        },
+        staticPreview: this.render({ version: 'v0.08' }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'opm',
+    }
+  }
+
+  static render({ version }) {
+    const color = '#599059'
+
+    return { message: addv(version), color }
```

Comment:
Would you explain the thinking for using a fixed color for the badge instead of the `renderVersionBadge` helper?

Reply:
I was copying from luarocks.service and found it has this fixed color behaviour. Is it a better practice to use renderVersionBadge? If so I can make the change.

Reply:
In general, I'd always prefer to use our helper functions to ensure that all of our badges within a given category are consistent. However, the Luarocks version badge does use a distinct/uncommon (at least within Shields) color scheme. 

I'm unclear on what the relationship is between OPM and Luarocks TBH, but I do think we should either use the common `renderVersionBadge` helper here, or the same coloring logic that is used for the Luarocks version badges if there's a strong connection/commonality between the two and users of OPM badges would expect the same coloring as the Luarocks badges. 

https://github.com/badges/shields/blob/98a12c97176650a27792749f5137aeba91321966/services/luarocks/luarocks.service.js#L48-L62

Reply:
👍 I've changed to use `renderVersionBadge` instead.